protocols/perf/src/client/handler.rs

```diff
@@ -186,18 +183,6 @@ impl ConnectionHandler for Handler {
             }
         }
 
-        if self.outbound.is_empty() {
-            match self.keep_alive {
-                KeepAlive::Yes => {
-                    self.keep_alive = KeepAlive::Until(Instant::now() + Duration::from_secs(10));
```

Comment:
Same here, we need to remove `KeepAlive::Until` first before we can change this.

Reply:
reverted b87b155